build.gradle

```diff
@@ -97,6 +97,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named('githubRelease') {
     dependsOn('createReleaseTag')
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
```

Comment:
I believe this line can be removed now that the new GH plugin release is out and applied to this build. Please verify by publishing a new version to the dev portal.